README.md

```diff
@@ -1 +1,163 @@
+
+
+## Community
+
+### What's Cozy?
+
+<div align="center">
+  <a href="https://cozy.io">
+    <img src="https://cdn.rawgit.com/cozy/cozy-site/master/src/images/cozy-logo-name-horizontal-blue.svg" alt="cozy" height="48" />
+  </a>
+ </div>
+ </br>
+
+[Cozy] is a platform that brings all your web services in the same private space.  With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
+
+### Localization
+
+Localization and translations are handled by [Transifex][tx], which is used by all Cozy's apps.
+
+As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim an access to the [app repository][tx-app]. Locales are pulled when app is build before publishing.
+
+As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
+
+
+### Maintainer
+
+The lead maintainer for Passwords is [cozy](https://github.com/cozy), send him/her a :beers: to say hello!
+
+
+### Get in touch
+
+You can reach the Cozy Community by:
+
+- Chatting with us on IRC [#cozycloud on Freenode][freenode]
+- Posting on our [Forum][forum]
+- Posting issues on the [Github repos][github]
+- Say Hi! on [Twitter][twitter]
+
+
+## License
+
+Passwords is developed by cozycloud and distributed under the [AGPL v3 license][agpl-3.0].
+
+
+
+[cozy]: https://cozy.io "Cozy Cloud"
+[setup]: https://dev.cozy.io/#set-up-the-development-environment "Cozy dev docs: Set up the Development Environment"
+[yarn]: https://yarnpkg.com/
+[yarn-install]: https://yarnpkg.com/en/docs/install
+[cozy-ui]: https://github.com/cozy/cozy-ui
+[cozy-client-js]: https://github.com/cozy/cozy-client-js/
+[cozy-stack-docker]: https://github.com/cozy/cozy-stack/blob/master/docs/client-app-dev.md#with-docker
+[doctypes]: https://cozy.github.io/cozy-doctypes/
+[bill-doctype]: https://github.com/cozy/cozy-konnector-libs/blob/master/models/bill.js
+[konnector-doctype]: https://github.com/cozy/cozy-konnector-libs/blob/master/models/base_model.js
+[konnectors]: https://github.com/cozy/cozy-konnector-libs
+[agpl-3.0]: https://www.gnu.org/licenses/agpl-3.0.html
+[contribute]: CONTRIBUTING.md
+[tx]: https://www.transifex.com/cozy/
+[tx-signin]: https://www.transifex.com/signin/
+[tx-app]: https://www.transifex.com/cozy/<SLUG_TX>/dashboard/
+[tx-client]: http://docs.transifex.com/client/
+[freenode]: http://webchat.freenode.net/?randomnick=1&channels=%23cozycloud&uio=d4
+[forum]: https://forum.cozy.io/
+[github]: https://github.com/cozy/
+[twitter]: https://twitter.com/cozycloud
+[nvm]: https://github.com/creationix/nvm
+[ndenv]: https://github.com/riywo/ndenv
+[jest]: https://facebook.github.io/jest/
```

Comment:
Nit: I would advise to remove completely this file, it is always an out of date copy paste of other files.